go.mod

```diff
@@ -75,7 +75,7 @@ require (
 	github.com/smartcontractkit/chain-selectors v1.0.23
 	github.com/smartcontractkit/chainlink-automation v1.0.4
 	github.com/smartcontractkit/chainlink-ccip v0.0.0-20240924115754-8858b0423283
-	github.com/smartcontractkit/chainlink-common v0.2.3-0.20240925150249-664dd5a59715
+	github.com/smartcontractkit/chainlink-common v0.2.3-0.20241016215635-229d99ed1e92
```

Comment:
This is head of https://github.com/smartcontractkit/chainlink-common/tree/release/2.17.1-backport, which is a backport of https://github.com/smartcontractkit/chainlink-common/pull/858 on top of `664dd5a59715` to minimize changes.